### ISSUE TITLE
Make it possible to configure HTTP_TIMEOUT const

### ISSUE THREAD

**Issue by yoprysk-dh-nf** [launchdarkly/terraform-provider-launchdarkly#174]
When performing parallel deployments for the same project but different environments(terraform-provider-launchdarkly version: 2.13.3), we start to see next errors:

```
Error: failed to update flag "feature-flag-name" in project "default": Patch "https://app.launchdarkly.com/api/v2/flags/default/feature-flag-namei": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```

Could you make HTTP_TIMEOUT configurable instead of hard coded value?
https://github.com/launchdarkly/terraform-provider-launchdarkly/blob/b34058fbc17076e14dd291bc16416001ff29cb7e/launchdarkly/config.go#L25

**Comment by ldhenry** [comment]
Hey @yoprysk-dh-nf,

This sounds like a reasonable request. We'll plan to include it with out next release. I'll let you know when it's available.

Thanks,
Henry

**Comment by ldhenry** [comment]
Hey @yoprysk-dh-nf , we just released v2.15.0 with a new [`http_timeout` provider](https://registry.terraform.io/providers/launchdarkly/launchdarkly/latest/docs#http_timeout) option.

I'm going to close this issue for now but feel free to reopen if you are still experiencing issues.

**Comment by yoprysk-dh-nf** [comment]
> Hey @yoprysk-dh-nf , we just released v2.15.0 with a new [`http_timeout` provider](https://registry.terraform.io/providers/launchdarkly/launchdarkly/latest/docs#http_timeout) option.
> 
> I'm going to close this issue for now but feel free to reopen if you are still experiencing issues.

Thanks a lot, for quick fix
